File: Java/1827_Minimum_Operations_to_Make_the_Array_Increasing.java
class Solution {
    public int minOperations(int[] nums) {
        int result = 0;
        
        for(int i = 1; i < nums.length; i++){
            if(nums[i] > nums[i-1]){
                continue;
            }
            else{
                result += (nums[i-1] - nums[i] + 1);
                nums[i] = nums[i-1]+1;
            }
        }
        return result;
    }
}

File: CS/1827_Minimum_Operations_to_Make_the_Array_Increasing.cs
public class Solution {
    public int MinOperations(int[] nums) {
        int result = 0;
        for(int i = 1; i < nums.Length; i++){
            if(nums[i-1] >= nums[i]){
                /*
                int temp = nums[i];
                nums[i] = nums[i-1] + 1;
                result += nums[i] - temp;
                */
                result += nums[i-1] + 1 - nums[i];
                nums[i] = nums[i-1] + 1;
            }
        }
        return result;
    }
}
